### PR TITLE
fix: camera add in builder in world enter/exit

### PR DESCRIPTION
**Pull request by D4rWiNSS**
#What? 
Now the BuilderInWorld activates/deactivates the additional camera and its components. Also, it only set the renderer when necessary

# Why? 
This PR solves the issue with the builder in world camera lowering the overall performance of the client

### CODE REVIEW

**Review by pravusjif — APPROVED** [review]
I did a bit of profiling and can confirm that the builder camera is no longer causing trouble as before

File: unity-client/Assets/Builder/Scripts/DCLBuilderOutline.cs [hunk 1]
```diff
@@ -54,6 +54,26 @@ private void OnDestroy()
...
         }
 
+        public void Activate()
+        {
+            outlineCamera.gameObject.SetActive(true);
+            outlineCanvas.gameObject.SetActive(true);
+            outlineRawImage.gameObject.SetActive(true);
+        }
+    
+        public void Desactivate()
```
Comment (BrianAmadori) [inline unity-client/Assets/Builder/Scripts/DCLBuilderOutline.cs]:
```suggestion
        public void Deactivate()
```

File: unity-client/Assets/Builder/Scripts/DCLBuilderOutline.cs [hunk 2]
```diff
@@ -54,6 +54,26 @@ private void OnDestroy()
...
+        {
+            outlineCamera.gameObject.SetActive(false);
+            outlineCanvas.gameObject.SetActive(false);
+            outlineRawImage.gameObject.SetActive(false);
+        }
+
+        public void SetOutlinerMaterial(Material mat)
```
Comment (BrianAmadori) [inline unity-client/Assets/Builder/Scripts/DCLBuilderOutline.cs]:
```suggestion
        public void SetOutlineMaterial(Material mat)
```

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuildModeController.cs [hunk 3]
```diff
@@ -655,15 +658,15 @@ public void EnterEditMode()
...
         CommonScriptableObjects.builderInWorldNotNecessaryUIVisibilityStatus.Set(false);
      
         DCLCharacterController.OnPositionSet += ExitAfterCharacterTeleport;
         builderInputWrapper.gameObject.SetActive(true);
         builderInWorldEntityHandler.EnterEditMode(sceneToEdit);
 
         SceneController.i.ActiveBuilderInWorldEditScene();
-     
+
+        ActiveBuilderInWorldCamera();
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuildModeController.cs]:
```suggestion
        ActivateBuilderInWorldCamera();
```

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuildModeController.cs [hunk 4]
```diff
@@ -680,21 +683,53 @@ public void ExitEditMode()
...
-           
+
+
         DCLCharacterController.OnPositionSet -= ExitAfterCharacterTeleport;
         builderInputWrapper.gameObject.SetActive(false);
         builderInWorldBridge.ExitKernelEditMode(sceneToEdit);
 
         SceneController.i.DesactiveBuilderInWorldEditScene();
+
+        DesactiveBuilderInWorldCamera();
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuildModeController.cs]:
```suggestion
        DeactivateBuilderInWorldCamera();
```

**Review by BrianAmadori — APPROVED** [review]
LGTM! Just left a few typo requests.

**Pull request merged**